numcodecs/json.py

```diff
@@ -55,15 +55,18 @@ def __init__(self, encoding='utf-8', skipkeys=False, ensure_ascii=True,
 
     def encode(self, buf):
         buf = np.asarray(buf)
-        items = buf.tolist()
+        items = np.atleast_1d(buf).tolist()
         items.append(buf.dtype.str)
         items.append(buf.shape)
```

Comment:
Should we store shape before converting to 1-D?

Reply:
buf has not been changed, just gone through asarray. Actually it wouldn't matter, since zarr will reshape to the output array anyway.